hack/ensure-kind.sh

```diff
@@ -46,7 +46,13 @@ verify_kind_version() {
   fi
 
   local kind_version
-  kind_version="v$(kind version -q)"
+  if [ -x "$(command -v kind)" ]; then
+    kind_version="v$(kind version -q)"
+  else
+    echo "warning: GOPATH_BIN=${GOPATH_BIN} not in your path"
+    kind_version="v$("${GOPATH_BIN}"/kind version -q)"
```

Comment:
Q: Should we fail in this case? I assume we are using kind command without path everywhere else...

Reply:
Good point. We should fail. Given the goal of this script is to make kind actually available

Reply:
True. Changed to printing an error and returning 2. Now the error is clear for the user and it ensures kind is usable without path.